index.html

```diff
@@ -1797,23 +1775,22 @@
                 </li>
                 <li>
                   <a>Queue a task</a> to <a>fire</a> an event named
-                  <code>connectionavailable</code> at <a data-lt=
-                  "PresentationReceiver"><code>presentation.receiver</code></a>.
+                  <code>connectionavailable</code> at all
```

Comment:
The idea is that there is only one list per receiver, yes. Is this a misconception?